polyply/src/minimizer.py

```diff
@@ -77,7 +77,12 @@ def compute_dih(params, coords):
     -------
     float
     """
-    # only optimize imporper dihedrals
+    # only optimize imporper dihedrals, because we don't
+    # treat multiplicity correctly; imporpers
+    # are needed for keeping conjugated systems falt;
+    # proper dihedrals will get adjusted in the energy
+    # minization
```

Comment:
This should go to the docstring, which is now very sparse/incorrect.

Reply:
btw, note the multiple *imporerp(s)* and *falt*